react/src/components/AssetDetail/AssetDetail.tsx

```diff
@@ -72,25 +81,16 @@ const AssetDetail: React.FC<AssetModalProps> = ({
         <Button type="link" iconNameAfter="close" onClick={onClose}></Button>
       </div>
       <div className={styles.middleSection}>
-        {fileType ? (
-          <>
-            <Suspense fallback={<LoadingSpinner />}>
-              <div className={styles.assetContainer}>
-                <AssetRenderer type={fileType} source={featureSource} />
-              </div>
-            </Suspense>
-            <Button /*TODO Download Action */>Download</Button>
-          </>
-        ) : (
-          <>
-            <SectionMessage type="info">Feature has no asset.</SectionMessage>
-            {!isPublicView && (
-              <Button type="primary" /* TODO Add asset to a feature */>
-                Add asset from DesignSafe
-              </Button>
-            )}
-          </>
-        )}
+        <Suspense fallback={<LoadingSpinner />}>
+          <div className={styles.assetContainer}>
```

Comment:
for point clouds, i think they need to take up more of the space.  and then for images, I see the download button appears in the middle before the image preview shows up.  i think if we added a width:100% and height:100% to assetContainer style that would fix those but then things would need to be tweaked so "This feature has no assets" has the right size.

Reply:
This is updated. I did something a bit different than your suggestion to avoid stretching horizontal pictures or videos. Let me know what you think!

Reply:
awesome! looks good.